backend/app/services/first_run_doctor.py

```
@@ -500,27 +512,163 @@ def _source_governance_unavailable_checks(
     )
 
 
-def _check_frontend_expectation(
+def _check_backend_health(
+    backend_base_url: str,
+    probe: Callable[[str], HttpProbeResponse],
+) -> FirstRunDoctorCheck:
+    health_url = f"{_normalize_base_url(backend_base_url)}/health"
+    try:
+        response = probe(health_url)
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)